pom.xml

```diff
@@ -201,10 +201,12 @@
     </dependencies>
```

Comment:
@btyoshikawaa 
It's necessary. So I added it. e4ccb8db8bcc30fd9f455b696bf13737aa083fef